### PR TITLE
Dynamically set the URL template based on the build env

### PR DESCRIPTION
**Pull request by davidcassany**
A follow up of #939

### CODE REVIEW

File: tests/cypress/latest/support/commands.ts [hunk 1]
```diff
@@ -388,7 +388,10 @@ Cypress.Commands.add('checkLabelSize', (sizeToCheck) => {
 
 // Add an OS version channel
 Cypress.Commands.add('addOsVersionChannel', (channelVersion) => {
-    const channelRepo = `registry.opensuse.org/isv/rancher/elemental/${channelVersion}/containers/rancher/elemental-teal-channel/5.4:latest`;
+    let channelRepo = `registry.opensuse.org/isv/rancher/elemental/${channelVersion}/containers/rancher/elemental-teal-channel/5.4:latest`;
```
Comment (kkaempf) [inline tests/cypress/latest/support/commands.ts:391]: Hmm, this still has an embedded version number (5.4) 🤔

**Review by kkaempf — APPROVED** [review]
worth a try 😆

File: tests/cypress/latest/support/commands.ts (hunk 1 above)
Comment (davidcassany) [reply tests/cypress/latest/support/commands.ts:391]: Yes, I already created a CI card to address that here #940 this is a more generic issue within the CI, there are many references out there to the SLE Micro version. They should be limited and parametrized, I'd prefer to have Loic and Juilen around to address this, they might have opinions and good ideas about how to handle this.

Comment (davidcassany) [reply tests/cypress/latest/support/commands.ts:391]: Also in relation to that there is also https://github.com/rancher/elemental-operator/pull/491 which is about removing SLE Micro version references from images that are not explicitly based on SLE Micro for rancher (all but the OS images)

**Pull request merged**